Templates/swiftui/Project+Templates.stencil

```diff
@@ -1,58 +1,47 @@
-                                                "UISceneDelegateClassName": "$(PRODUCT_MODULE_NAME).SceneDelegate"
-                                            ]
-                                        ]
-                                    ]
-                                ]])
+extension Project {
+    /// Helper function to creates the Project for this ExampleApp
+    public static func app(name: String, additionalTargets: [Target]) -> Project {
+        var targets = makeAppTargets(name: name, dependencies: additionalTargets.map { TargetDependency.target(name: $0.name) })
+        targets += additionalTargets
+        return Project(name: name,
+                       organizationName: "tuist.io",
+                       targets: targets)
     }
 
-    public static func framework(name: String, platform: Platform, dependencies: [TargetDependency] = []) -> Project {
-        return self.project(name: name, product: .framework, platform: platform, dependencies: dependencies)
-    }
+    /// Helper function to create the application target and the unit test target for the ExampleApp
+    private static func makeAppTargets(name: String, dependencies: [TargetDependency]) -> [Target] {
+        let platform: Platform = .{{ platform }}
+        let infoPlist: [String: InfoPlist.Value] = [
+            "CFBundleShortVersionString": "1.0",
+            "CFBundleVersion": "1",
+            {% if platform == "macOS" %}"NSMainStoryboardFile"{% else %}"UIMainStoryboardFile"{% endif %}: ""
+            ]
 
-    public static func project(name: String,
-                               product: Product,
-                               platform: Platform,
-                               dependencies: [TargetDependency] = [],
-                               infoPlist: [String: InfoPlist.Value] = [:]) -> Project {
-        return Project(name: name,
-                       targets: [
-                        Target(name: name,
-                                platform: platform,
-                                product: product,
-                                bundleId: "io.tuist.\(name)",
-                                infoPlist: .extendingDefault(with: infoPlist),
-                                sources: ["Sources/**"],
-                                resources: [],
-                                dependencies: dependencies),
-                        Target(name: "\(name)Tests",
-                                platform: platform,
-                                product: .unitTests,
-                                bundleId: "io.tuist.\(name)Tests",
-                                infoPlist: .default,
-                                sources: "Tests/**",
-                                dependencies: [
-                                    .target(name: "\(name)")
-                                ])
-                      ])
-    }
+        let mainTarget =  Target(name: name,
```

Comment:
there seems to be an extra space between `=` and `Target`. Also the indentation here does not seem right